client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -81,77 +81,69 @@ class DisconnectJetpack extends PureComponent {
-					translate(
-						'{{icon/}} Daily automated backups (unlimited storage)',
-						this.getIcon( 'history' )
-					)
-				);
-				features.push(
-					translate( '{{icon/}} Daily automated malware scanning', this.getIcon( 'spam' ) )
-				);
-				features.push(
-					translate( '{{icon/}} Priority WordPress and security support', this.getIcon( 'chat' ) )
-				);
-				features.push(
-					translate( '{{icon/}} 13GB of high-speed video hosting', this.getIcon( 'video' ) )
-				);
-				break;
 
-			case 'is-business-plan':
-				features.push(
-					translate(
-						'{{icon/}} Daily automated backups (unlimited storage)',
-						this.getIcon( 'history' )
-					)
-				);
-				features.push(
-					translate(
-						'{{icon/}} Daily automated malware scanning with automated resolution',
-						this.getIcon( 'spam' )
-					)
-				);
-				features.push(
-					translate( '{{icon/}} Priority WordPress and security support', this.getIcon( 'chat' ) )
-				);
-				features.push(
-					translate( '{{icon/}} Unlimited high-speed video hosting', this.getIcon( 'video' ) )
-				);
-				features.push( translate( '{{icon/}} SEO preview tools', this.getIcon( 'globe' ) ) );
-				break;
+		if ( plan === 'is-free-plan' ) {
+			features.push(
+				translate(
+					'{{icon/}} Site stats, related content, and sharing tools',
+					this.getIcon( 'stats-alt' )
+				)
+			);
+			features.push(
+				translate(
+					'{{icon/}} Brute force attack protection and downtime monitoring',
+					this.getIcon( 'lock' )
+				)
+			);
+			features.push(
+				translate( '{{icon/}} Unlimited, high-speed image hosting', this.getIcon( 'image' ) )
+			);
+		} else if ( plan === 'is-personal-plan' ) {
+			features.push(
+				translate(
+					'{{icon/}} Daily automated backups (unlimited storage)',
+					this.getIcon( 'history' )
+				)
+			);
+			features.push(
+				translate( '{{icon/}} Priority WordPress and security support', this.getIcon( 'chat' ) )
+			);
+			features.push( translate( '{{icon/}} Spam filtering', this.getIcon( 'spam' ) ) );
+		} else if ( [ 'is-premium-plan', 'is-daily-security-plan' ].includes( plan ) ) {
+			features.push(
+				translate(
+					'{{icon/}} Daily automated backups (unlimited storage)',
+					this.getIcon( 'history' )
+				)
+			);
+			features.push(
+				translate( '{{icon/}} Daily automated malware scanning', this.getIcon( 'spam' ) )
+			);
+			features.push(
+				translate( '{{icon/}} Priority WordPress and security support', this.getIcon( 'chat' ) )
+			);
+			features.push(
+				translate( '{{icon/}} 13GB of high-speed video hosting', this.getIcon( 'video' ) )
+			);
+		} else if (
+			[ 'is-business-plan', 'is-realtime-security-plan', 'is-complete-plan' ].includes( plan )
+		) {
+			features.push(
+				translate(
+					'{{icon/}} Real-time automated backups (unlimited storage)',
+					this.getIcon( 'history' )
+				)
+			);
+			features.push(
+				translate( '{{icon/}} Real-time automated malware scanning', this.getIcon( 'spam' ) )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( '{{icon/}} Daily automated malware scanning' )` *ES Score: 7*